draft-ietf-tls-tls13.md

```diff
@@ -1825,7 +1825,7 @@ is compromised and client authentication is used, then the attacker
 can impersonate the client on subsequent connections.
 
 
-### Resumption and PSK
+### Resumption and Pre-Shared Key (PSK)
```

Comment:
Changing this actually breaks the auto-generated anchor links. Rather than make the ref longer, please append to this line the following:
" {#resumption-and-psk}"
(without the quotes, but with the leading space after the new text; note, only 1 set of braces)

This will keep the same anchor name whilst allowing the title to be changed.